.drone.yml

```diff
@@ -3,9 +3,11 @@
 matrix:
   SBT_RUN:
     - sbt "benchmarks/compile" \
-          "millBloop/compile" \
           "jsonConfig210/test" \
           "jsonConfig212/test" \
+          "millBloop/compile" \
+          "mavenBloop/compile" \
+          "gradleBloop/compile" \
```

Comment:
Can we change this to `gradleBloop/test` to test the new gradle plugins in the CI?